inception-active-learning/src/main/java/de/tudarmstadt/ukp/inception/active/learning/sidebar/ActiveLearningRecommender.java

```diff
@@ -380,8 +407,9 @@ public AnnotationObject generateRecommendationWithLowestConfidence(JCas aJcas)
         if (recommendations.size() > 0) {
             return recommendations.get(0);
         }
-        else
+        else {
             return null;
+        }
```

Comment:
Consider using `Optional.of(recommendations.get(0)` or even `recommendations.stream().findFirst()` which would allow for omitting the check if the collection is empty.

Reply:
have changed to use Optional to omit check if the collection is empty

Reply:
I think in my case, Optional is not the most necessary. I don't want to avoid null because AL sidebar needs it to check whether to show the suggestion or no suggestion label. 

I can also use Optional.isPresent to check, but then there is a serializable problem if I use it in AL sidebar.